pkg/installer/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Dockerfile to build installer img initrd
+FROM lfedge/eve-alpine:1f7685f95a475c6bbe682f0b976f12180b6c8726 AS build
+SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
+ENV BUILD_PKGS grep patch go git make gcc linux-headers musl-dev autoconf automake pkgconfig kmod-dev util-linux-dev cryptsetup-dev lddtree libgcc
+ENV PKGS mtools dosfstools libarchive-tools sgdisk e2fsprogs util-linux squashfs-tools coreutils tar dmidecode \
+    kmod-libs cryptsetup-libs libblkid
+RUN eve-alpine-deploy.sh
+
+COPY . .
```

Comment:
Please don't copy the `Dockerfile` into the container

Reply:
This is all early draft, and will change a lot in the coming days. Don't waste your time on it yet @christoph-zededa

Reply:
But since you raised it, why not? The final image is `FROM scratch` anyways?